source/mapping-rules/field-customizations/edit-field-customizations.txt

```diff
@@ -0,0 +1,56 @@
+.. _rm-edit-customized-fields:
+
+======================
+Edit Customized Fields
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You have the option to edit customized fields. Consider 
+making edits to field customizations to accommodate specific 
+requirements such as indexing, schema adjustments, or field renaming.
+
+About this Task
+---------------
+
+If you edit a customized field data type and you haven't enabled the 
```

Comment:
Not sure if we need to mention this, I tested the feature and was not sure if the new fields would be appended or overwritten when changing types and re-running. I added this for clarity.